faucet/valves_manager.py

```diff
@@ -280,7 +280,7 @@ def request_reload_configs(self, now, new_config_file, delete_dp=None):
         if self.config_watcher.content_changed(new_config_file):
             self.logger.info('configuration %s changed, analyzing differences', new_config_file)
             result = self.load_configs(now, new_config_file, delete_dp=delete_dp)
-            self._notify({'CONFIG_CHANGE': {'success': result}})
+            self._notify({'CONFIG_CHANGE': {'success': result, 'new_config':self.meta_dp_state.top_conf.get('dps')}})
```

Comment:
don't prefix with new, and imply that it's dps... 'dps_config'

Reply:
Resolved. PTAL